backend/src/main/java/com/bakdata/conquery/models/datasets/concepts/MatchingStats.java

```diff
@@ -99,32 +99,32 @@ public CDateRange getSpan() {
 			);
 		}
 
-        public void addEvent(Table table, Bucket bucket, int event, String entityForEvent) {
-            numberOfEvents++;
-            if (foundEntities.add(entityForEvent)) {
-                numberOfEntities++;
-            }
+		public void addEvent(Table table, Bucket bucket, int event, String entityForEvent) {
```

Comment:
vermutlich sollte das jetzt in die UpdateMatchingStats verschoben werden, wo die Klasse mehrere konsumenten hat?

Reply:
Die Methode wird in `UpdateMatchingStatsMessage` und `MatchingStatsTests` benutzt.